ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -70,15 +71,25 @@ internal fun Markdown(
     textAlign: TextAlign? = null,
     allowLinks: Boolean = true,
 ) {
-    val root = parser.parse(text) as Document
+    if (LocalInspectionMode.current) {
+        Text(
+            text = text,
+            style = style,
+            color = color,
+            fontWeight = fontWeight,
+            textAlign = textAlign,
+        )
+    } else {
+        val root = parser.parse(text) as Document
 
-    val density = LocalDensity.current
-    val paragraphPadding = with(density) { style.lineHeight.toDp() }
+        val density = LocalDensity.current
+        val paragraphPadding = with(density) { style.lineHeight.toDp() }
```

Comment:
This was failing during previews because it seems it was passing a non-sp unit, which can't be converted to `dp`. I opted to just use a normal text composable for previews, but we can dedicate some more time in the future to figure out what's wrong if needed.